pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -76,6 +75,9 @@ import (
 	openapigenerated "github.com/openshift/origin/pkg/openapi"
 	securityapi "github.com/openshift/origin/pkg/security/apis/security"
 	"github.com/openshift/origin/pkg/version"
+
+	// TODO fix this install, it is required for TestPreferredGroupVersions to pass
+	_ "github.com/openshift/origin/pkg/authorization/apis/authorization/install"
```

Comment:
I added this install import per @deads2k recommendation to make `TestPreferredGroupVersions` pass.

Re-adding LGTM.